media-proxy/include/session-base.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __SESSION_BASE_H
+#define __SESSION_BASE_H
+
+#include <limits.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <atomic>
+#include <thread>
+#include <stdexcept>
+
+using namespace std; // for atomic_bool
```

Comment:
first you write `using namespace std;` and after that sometimes in code there is `std::` used. Let's unify approach. 

I would explicitly use `std::`, it makes obvious that given function is standard one.

Reply:
Agree, then this change also apply for atomic_bool

Reply:
yeah, I hope it is possible to do without breaking C code.

Reply:
Yes, I can do that

Reply:
done